docs/docs/quick-start.md

```diff
@@ -38,6 +38,18 @@ Try editing the home page in `src/pages/index.js`. Saved changes will live reloa
 
 ## What's next?
 
+### Use flags
+
+The CLI also supports two flags:
+
+- `-y` skips the questionnaire
+- `-ts` initializes your project with the [minimal TypeScript starter](https://github.com/gatsbyjs/gatsby-starter-minimal) instead of the [minimal JavaScript starter](https://github.com/gatsbyjs/gatsby-starter-minimal)
```

Comment:
```suggestion
- `-ts` initializes your project with the [minimal TypeScript starter](https://github.com/gatsbyjs/gatsby-starter-minimal-ts) instead of the [minimal JavaScript starter](https://github.com/gatsbyjs/gatsby-starter-minimal)
```